operator/pkg/legacy/secrets_test.go

```diff
@@ -0,0 +1,119 @@
+package legacy
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/chartutil"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func Test_injector_Enrich(t *testing.T) {
+	tests := map[string]struct {
+		existingSecrets []string
+		commonKey       string
+		commonSecrets   []string
+		secretMap       map[string][]string
+		vals            chartutil.Values
+		want            chartutil.Values
+		wantErr         error
+	}{
+		"empty": {},
```

Comment:
what for?

Reply:
Just a corner case to ensure no nil pointers, etc.